src/EntryList.js

```diff
@@ -0,0 +1,19 @@
+function EntryList() {
```

Comment:
EntryList should accept `itmes` as props.

`items` is an array of strings.

EntryList should render the content of the `items` array in `span` tag 

Please remove your changes from EntryList  and ensure to follow my instructions 
I mean remove this code 
```html
<div>
            <h6 className="yu" style="color:blue;">My List</h6>
            <form>
                <label>
                    <input type="text" placeholder="Enter Something"> </input>
                </label>
            </form>
            <div class="w3-padding w3-xlarge w3-teal">
                <i class="glyphicon glyphicon-trash"></i>

            </div>
        </div>
```

Reply:
please why should ?
I want to see if I can do the job without using props
how did you know the items are strings ?
why not div tag ?

Reply:
> please why should ? I want to see if I can do the job without using props how did you know the items are strings ? why not div tag ?

I think you didn't understand the requirement.

I want the EntryList to display list of items 


for example if I have this array of data `["one", "two", "three"]`

I want to be able to use EntryList to display the data above 

in the same time, I want EntryList to be very flexible and can be used to show any list of items 

for example 

it should be able to use to it to show this list also 

- Apples
- Bananas 
- Eggs 

 In short, I want it to be reusable 
what need to be done to the EntryList ?

Reply:
No problem 
But I wanted first to do the job without using the props 
Later with it
Just I wanted to see what is the difference

Reply:
You sent me this 
![A53195F2-C53B-43BE-A98E-D8C20A8C931E](https://user-images.githubusercontent.com/60723932/173237647-82685bdc-ed75-4027-ad66-1342bf014fce.png)

So how you want the items in your second example 
All the list in one box 
Or each item ( fruit 🍌  🍉 ) in one box ?!

2) why you did not accept table codes and also wanted me to remove html form codes 
So what kind of codes you want 
Thanks